app/src/main/java/edu/wpi/first/shuffleboard/app/widget/StockWidgets.java

```diff
@@ -30,6 +33,14 @@ public static void init() {
                .map(c -> (Class<Widget>) c)
                .filter(c -> c.isAnnotationPresent(Description.class))
                .forEach(Widgets::register);
+      Widgets.setDefaultWidget(DataTypes.String, "Text View");
+      Widgets.setDefaultWidget(DataTypes.Boolean, "Boolean Box");
+      Widgets.setDefaultWidget(DataTypes.Number, "Number Slider");
+      Widgets.setDefaultWidget(DataTypes.Map, "Network Table Tree");
+      Widgets.setDefaultWidget(DataTypes.SendableChooser, "ComboBox Chooser");
+      Widgets.setDefaultWidget(DataTypes.Encoder, "Encoder");
+      Widgets.setDefaultWidget(DataTypes.AnalogInput, "Voltage View");
+      Widgets.setDefaultWidget(DataTypes.SpeedController, "Speed Controller");
```

Comment:
We should use the overload of this method that takes a WidgetType rather than a String